css/main.css

```diff
@@ -30,3 +30,167 @@ header a:hover{
 .pull{
   float: right;
 }
+/* Cover Starts Here -------------------------------------------------------- */
+.cover{
+  background-image: url(../img/cover.jpg);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
```

Comment:
@ajinkyaa @swapnil2308 
Instead of using multiple properties use shorthand prperties just like here,
background having 4 properties instead of it you can add app property into one background property.